test/unit/alphabet/cigar/cigar_op_test.cpp

```diff
@@ -0,0 +1,39 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include "../alphabet_test_template.hpp"
+#include "../alphabet_constexpr_test_template.hpp"
+#include <seqan3/alphabet/cigar/cigar_op.hpp>
+
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet, cigar_op);
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet_constexpr, cigar_op);
+
+TEST(cigar_op, to_char_assign_char)
+{
+	for (char chr : std::string{"MDISHNPX="})
+    	EXPECT_EQ(to_char(cigar_op{}.assign_char(chr)), chr);
```

Comment:
Never use tabulator! We require spaces for indentation.

Reply:
Yeah, my editor randomly decides to put tabs sometimes. I fixed all occurrences that I saw!